src/python/bot/tasks/train_gradientfuzz_task.py

```diff
@@ -13,9 +13,297 @@
 # limitations under the License.
 """GradientFuzz training task."""
 
+from builtins import str
+
+import glob
+import os
+import shutil
+import sys
+
+from bot.fuzzers.ml.gradientfuzz import constants
+from bot.fuzzers.ml.gradientfuzz import run_constants
+from build_management import build_manager
+from fuzzing import corpus_manager
+from google_cloud_utils import storage
 from metrics import logs
+from system import archive
+from system import environment
+from system import new_process
+from system import shell
+
+# Model script directory.
+GRADIENTFUZZ_SCRIPTS_DIR = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), 'fuzzers', 'ml', 'gradientfuzz')
+
+
+def get_corpus(corpus_directory, fuzzer_name):
+  """Get corpus directory.
+
+  This function will download latest corpus backup file from GCS, unzip
+  the file and put them in corpus directory.
+
+  TODO(ryancao): Make default corpus directory.
```

Comment:
What's needed for this? I don't quite understand this TODO.

Reply:
Ah. I think I was making a note to myself to download the corpus backups into a predefined corpus directory. I'll get rid of the TODO.